keps/4136-admission-fair-sharing/README.md

```diff
@@ -190,6 +191,22 @@ within AdmissionScope and then other mechanisms are applied as usual.
 workloads based on the LocalQueue resource usage when considering workloads within the same ClusterQueue. 
 Workloads from different ClusterQueues are not compared against each other using the resource usage dimension.
 
+### Entry penalty
+Because of the interval between updating FairSharingStatus the mechanism can be exploited. We may end up in a situation where one tenant
+can submit thousands of jobs which will be prioritized because they had slightly lower FairSharingStatus. 
+
+E.g. Let's assume:
+```
+Tenant's A FairSharingStatus: 0.1 CPU
+Tenant's B FairSharingStatus: 0.05 CPU
+usageSamplingInterval: 5mins
+```
+
+In the worst case scenario all of the jobs submitted by Tenant B will be prioritized and could be admitted for the next 5mins.
+Even if Tenant B submitted a job that consumed 1000 CPUs, consecutive jobs would still be prioritized because of the delay in updating the status.
+
+Hence, since v0.13 Kueue adds an entry penalty to FairSharingStatus every time it admits a Workload. The penalty should be calculated with similar formula: `penalty = A * requested_resource`. We assume the amount of time had passed since the last update is equal to the `usageSamplingInterval`.
```

Comment:
Why this particular formula, is there some motivation / intuition behind it?

Reply:
Is the actual interpretation of the penalty: "usage which would have been added if the Job started in the previous sampling interval"? This feels arbitrary, why not "two sampling intervals ago", or any custom.

Maybe we should have a dedicated API param like "entryPenaltyMultiplier" instead in the future. However, I prefer to be conservative about the API, so I'm happy to start with the proposed penalty, but add a note along the lines: the choice entry penalty is arbitrary, but we propose it for alpha to be conservative about API. We may re-evaluate the need to make the penalty configurable based on user-feedback.

Reply:
> Is the actual interpretation of the penalty: "usage which would have been added if the Job started in the previous sampling interval"?

Exactly

> Maybe we should have a dedicated API param like "entryPenaltyMultiplier" instead in the future. However, I prefer to be conservative about the API, so I'm happy to start with the proposed penalty, but add a note along the lines: the choice entry penalty is arbitrary, but we propose it for alpha to be conservative about API. We may re-evaluate the need to make the penalty configurable based on user-feedback.

It's quite arbitrary indeed. I like the idea of possible API in the future, and I'm happy to extend it after we collect users' feedback